### ISSUE TITLE
Feedback wanted on tutorial idea

### ISSUE THREAD

**Issue by jdent02** [bsavery/PRMan-for-Blender#174]
Given the questions many (myself included) have about the different render settings, I was thinking I might try to create a Renderman RIS version of the Arnold reducing noise tutorial (https://support.solidangle.com/display/AFCUG/Removing+Noise).  The plan would be to step through each of the settings and just see (and document) what they do.  Would something like that be of value?

**Comment by bsavery** [comment]
Yes. Phillipe had done something previously for RenderMan for Maya.  But that would be a great idea!

**Comment by jdent02** [comment]
Any chance you could send me a link for that one?

Sent from my iPhone

On Jan 6, 2016, at 11:59 AM, bsavery <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes. Phillipe had done something previously for RenderMan for Maya. But that would be a great idea!

-

**Comment by tobbew79** [comment]
Personally if there is one for maya I'm quite certain I can extrapolate from that, the settings are documentet in the pr-documentation (I assume they are the same). The blender addon however is not very well documentet, and it would be a great help for me if you wrote (not a video) about current workarounds and advice on different settings that might be hard to find in the ui-design and kept it up to date.

**Comment by jdent02** [comment]
I haven't had any luck so far in tracking down the Maya tutorial Brian mentioned (or any other in depth tutorial on noise reduction workflows for RIS), so that's one of the reasons I want to create one myself from scratch and post it here.

**Comment by pleprince** [comment]
I haven't written it yet, but I have been explaining the basics on the forum for a while...
https://renderman.pixar.com/forum/showthread.php?s=&postid=169638#post169638
https://renderman.pixar.com/forum/showthread.php?s=&postid=175123#post175123
and others...

**Comment by jdent02** [comment]
Pleprince that's exactly the type of stuff that I think more people need to see (especially casual users who may not be all that familiar with path tracing innards).  However, I think there's still something missing, and that is the bigger picture of why your suggesting these settings.  How is PRman really working?  What, in layman's terms, do these render settings do?  That Arnold doc was a breakthrough for me in understanding how to tweak a path tracer, not only because it told you what to do to solve noise problems, but it also told you WHY it solved the problem, and I was hoping that something similar could be put here so people don't have to dig through the forums or spend time asking the questions.  Tell the driver how their engine is working (with pictures) and they can come up with their own solutions.

If either you or tobbew79 have stuff in the works then by all means let me know and I'll tackle something else, otherwise I'll still plan on drafting something up over the next few weeks.

**Comment by pleprince** [comment]
That's the plan...
1 - how the path tracer works. keep it simple.
2 - how to optimise an outdoor scene
3 - how to optimise an indoor scene.
A lot of people are trying to use PxrPathTracer's "manual mode", but the "bxdf mode" works much better when you understand how our variance-driven sampling.

**Comment by jdent02** [comment]
I look forward to seeing that doc when it's up.  Any idea where it would be posted when it is?

And I didn't notice at first that you were a Pixar guy, so I hope I didn't stick my foot in my mouth too bad.  I'll close this issue out.
